changelog/changes/tried-series-night.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Add an option to add extra headers to the platform request"
+type: feature
+authors: lava
+pr: 5287
+---
+
+The new option `tenzir.platform-extra-headers` causes the Tenzir Node to add the given extra HTTP headers when
+establishing the connection to the Tenzir Platform.
```

Comment:
Why would I want this? Can we please describe a use case so that someone can understand why that feature is useful in the first place?